scripts/lib/package_managers.sh

```diff
@@ -18,8 +18,19 @@ gem_install() {
   gem install "$@" --no-document --quiet
 }
 
+brew_update() {
+  tmp_dir="$(mktemp -d)"
+  brew --version
+  brew update >> "$tmp_dir/brew_update_logs"
```

Comment:
Since we can't recover these files anyway, should they just be sent to `/dev/null`?